app/bundles/ChannelBundle/EventListener/SearchSubscriber.php

```diff
@@ -31,6 +33,24 @@ public static function getSubscribedEvents(): array
     }
 
     public function onGlobalSearch(GlobalSearchEvent $event): void
+    {
+        $searchString = $event->getSearchString();
+        if (empty($searchString)) {
+            return;
+        }
```

Comment:
Can this check be moved to the new service? Otherwise every subscriber will have to check for this.